cpp/include/Ice/Ice.h

```diff
@@ -22,7 +22,6 @@
 
 #    include "Communicator.h"
 #    include "Connection.h"
-#    include "IconvStringConverter.h"
```

Comment:
Alternatively we can keep the header and add a macro to control the inclusion

Reply:
I prefer to remove it from Ice.h like you did.